interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -61,656 +60,645 @@
+                }
             }
+            
+            $fields['RXTYPE'] = $RXTYPE;
+            $fields['REFDATE'] = $data['date'];
+            $insert_this_id = formSubmit($table_name, $fields, $form_id, $_SESSION['userauthorized']);
         }
-
-        $fields['RXTYPE'] = $RXTYPE;
-        $fields['REFDATE'] = $data['date'];
-        $insert_this_id = formSubmit($table_name, $fields, $form_id, $_SESSION['userauthorized']);
     }
-}
-
-if ($_REQUEST['dispensed']) {
-    $query = "SELECT * from form_eye_mag_dispense where pid =? ORDER BY date DESC";
-    $dispensed = sqlStatement($query, array($_REQUEST['pid']));
-    ?><html>
+   // $GLOBALS['assets_static_relative']= "/openemr/public/assets";
+    
+    if ($_REQUEST['dispensed']) {
+        $query = "SELECT * from form_eye_mag_dispense where pid =? ORDER BY date DESC";
+        $dispensed = sqlStatement($query, array($_REQUEST['pid']));
+        ?><html>
     <title><?php echo xlt('Rx Dispensed History'); ?></title>
     <head>
-    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-1-10-2/jquery.js"></script>
-    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap/dist/js/bootstrap.min.js"></script>
-
-    <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
-
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['css_header']; ?>" type="text/css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-themes-1-10-4/themes/ui-lightness/jquery-ui.min.css">
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/pure/0.5.0/pure-min.css">
-
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/font-awesome/css/font-awesome.min.css">
-    <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css" type="text/css">
 
-    <style>
-        .title {
-            font-size:1em;
-            position:absolute;
-            right:10px;
-            top:30px;
-            font-size: 1em;
-        }
-        .refraction {
-            width:95%;
-            display: -moz-stack;
-            vertical-align: middle;
-            min-height:unset;
-        }
-        .refraction td {
-            text-align:center;
-            font-size:12px;
-            width:0.9in;
-            vertical-align: text-middle;
-            text-decoration: unset;
-        }
-        table {
-            font-size: 1.0em;
-            padding: 12px;
-            color: black;
-            vertical-align: text-top;
-        }
+        <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
+        <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','pure', 'bootstrap' ,'jscolor' ]); ?>
```

Comment:
bootstrap can be remove from above since it is always included
can also add 'opener' in line above and then remove the line at 291 (include_opener.js include)
you should also be able to remove line 294 (css_header include) since this is always included in above line

Reply:
done